doc/nrf/ug_thingy53.rst

```diff
@@ -23,15 +23,14 @@ The |NCS| provides support for developing applications on the Thingy:53.
 Connecting to Thingy:53 serial ports
```

Comment:
Not related to any change in this PR, but I missed the TYPO in the original PR: please modify in line 14: IEEE 802.14.5 -> IEEE 802.**15.4**

Reply:
Fixed